services/registrationNotification.js

```diff
@@ -1,22 +1,12 @@
-const nodemailer = require("nodemailer");
 require('dotenv').config();
 
-exports.sendEmail = (user) => {
-    var transporter = nodemailer.createTransport({
-        service: 'gmail',
-        auth: {
-            user: process.env.USER_EMAIL,
-            pass: process.env.USER_PASS,
-        }
-    });
-
-    var mailOptions = {
+exports.sendNotif = (user) => {
+    const mailOptions = {
         from: '"Sport News" <process.env.USER_EMAIL>',
         to: user,
         subject: 'Subscription in Sport News',
         html: `<table style="background:#f9f9fb;width: 100%;padding: 0 20%;">
```

Comment:
pls move template to some template directory with registrationNotificationTemplate name